LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -76,11 +77,18 @@ public IActionResult Index(Player player)
 
 
             //TODO: Step 1
+            if (currentBalance < 0.5m)
+            {
+                return RedirectToAction("LuckList");
+            }
 
-
+            
+            else
+            {
+                currentBalance -= .5m;
+            }
             //TODO: Step 2
-
-
+            if (SpinBalance.)
```

Comment:
To check the player's spin, use the Spin object from line 57 `if(spin.IsWinning) { ...}`